src/reconcile/rec_row.c

```diff
@@ -467,25 +466,42 @@ __wt_rec_row_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
             key->cell_len = 0;
             key->len = key->buf.size;
             ovfl_key = true;
+            page_image += ikey->size;
```

Comment:
`page_size` only account for the key size, that too only when overflow key is involved. Is that intentional. Maybe we can rename the variable to reflect that

Reply:
Renamed to `key_overflow_size` in 08dab949c.